src/main/kotlin/pl/edu/agh/server/infrastructure/DataLoader.kt

```diff
@@ -726,4 +789,12 @@ class DataLoader(
     private fun shortLoremIpsumPl(): String {
         return "Ale muszę wam wytłumaczyć, jak narodziła się ta błędna koncepcja denuncjacji przyjemności i chwalebnego bólu, a ja dam wam kompletną relację z systemu i objaśnię prawdziwe nauki wielkiego odkrywcy prawdy, mistrza-budowniczego ludzkiego szczęścia. Nikt nie odrzuca, nie lubi lub unika przyjemności samej w sobie."
     }
+
+    private fun assignRoles() {
+        val admin = userRepository.findByEmail("admin@agh.edu.pl").get()
+        organizationRepository.findAll().forEach { organization ->
+            organizationService.assignUserRole(organization.id!!, admin.id!!, OrganizationRole.CONTENT_CREATOR)
+            organizationService.assignUserRole(organization.id!!, admin.id!!, OrganizationRole.HEAD)
+        }
+    }
```

Comment:
Z tego rozumiem, ze kazdy admin musi miec zawsze wszystkie prawa w kazdej organizacji. Czyli przy tworzeniu organizacji trzeba bedzie wyciagac i modyfikowac wszystkich adminow, a przy tworzeniu admina trzeba bedzie iterowac po wszystkich organizacjach.

Nie wolimy zamiast tego w tych dekoratorach zrobic dodatkowego sprawdzenia i roli ADMIN, zeby dodwac ADMIN poprostu w tej liscie dozwolonych rol?

Reply:
faktycznie, nie pomyślałem o tym rozwiązaniu a zapomniałem że moge skipnąć jeśli jest rola admin jaka to organizacja